packages/frontend/core/src/router.tsx

```diff
@@ -1,59 +1,81 @@
-import * as Sentry from '@sentry/react';
+import { wrapCreateBrowserRouter } from '@sentry/react';
+import { useEffect } from 'react';
 import type { RouteObject } from 'react-router-dom';
-import { createBrowserRouter as reactRouterCreateBrowserRouter } from 'react-router-dom';
+import {
+  createBrowserRouter as reactRouterCreateBrowserRouter,
+  Outlet,
+  useLocation,
+} from 'react-router-dom';
 
-export const workbenchRoutes = [
-  {
-    path: '/',
-    lazy: () => import('./pages/index'),
-  },
-  {
-    path: '/workspace/:workspaceId/*',
-    lazy: () => import('./pages/workspace/index'),
-  },
-  {
-    path: '/share/:workspaceId/:pageId',
-    lazy: () => import('./pages/share/share-detail-page'),
-  },
-  {
-    path: '/404',
-    lazy: () => import('./pages/404'),
-  },
-  {
-    path: '/auth/:authType',
-    lazy: () => import('./pages/auth'),
-  },
-  {
-    path: '/expired',
-    lazy: () => import('./pages/expired'),
-  },
-  {
-    path: '/invite/:inviteId',
-    lazy: () => import('./pages/invite'),
-  },
-  {
-    path: '/signIn',
-    lazy: () => import('./pages/sign-in'),
-  },
-  {
-    path: '/open-app/:action',
-    lazy: () => import('./pages/open-app'),
-  },
-  {
-    path: '/upgrade-success',
-    lazy: () => import('./pages/upgrade-success'),
-  },
-  {
-    path: '/desktop-signin',
-    lazy: () => import('./pages/desktop-signin'),
-  },
-  {
-    path: '/onboarding',
-    lazy: () => import('./pages/onboarding'),
-  },
-  {
-    path: '*',
-    lazy: () => import('./pages/404'),
+import { mixpanel } from './utils';
+
+function RootRouter() {
+  const location = useLocation();
+  useEffect(() => {
+    mixpanel.track_pageview({
```

Comment:
looks like it will reports error if mixpanel is not enabled. maybe we need to check MIXPANEL_TOKEN env before calling this method